src/components/Calculator.js

```diff
@@ -0,0 +1,95 @@
+/* eslint jsx-quotes: ["error", "prefer-single"] */
+import React, { Component } from 'react';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className='calculatorContainer'>
+        <div className='calculatorResult'>0</div>
+        <div className='calculatorButtonsContainer'>
+          <div className='numbersGroupContainer'>
+            <div className='numbersGroup'>
+              <button className='button' type='button'>
+                AC
+              </button>
+              <button className='button' type='button'>
+                +/-
+              </button>
+              <button className='button' type='button'>
+                %
+              </button>
+            </div>
+            <div className='numbersGroup'>
+              <button className='button' type='button'>
+                7
```

Comment:
- [OPTIONAL] 🤔 This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻